docs/best-practices/data-import.rst

```diff
@@ -240,17 +250,18 @@ will return successfully, reporting ``COPY OK, 0 rows affected (... sec)``.
 
 .. NOTE::
 
-  When importing data using ``COPY FROM`` CrateDB does not check if the types
-  from the columns and the types from the import file match. It does not cast
-  the types to their target but will always import the data as in the source
-  file(s).
+  When importing data using ``COPY FROM``, CrateDB does not check whether the
+  types from the columns and the types from the import file match. It does not
+  cast the types to their target but will always import the data as in the
+  source file(s).
```

Comment:
"(s)" strikes me as bad style

(I realize you didn't change anything to do with the wording here but it jumped out at me)